src/main/java/uk/gov/companieshouse/psc/delta/mapper/PscMapper.java

```diff
@@ -0,0 +1,223 @@
+                break;
+            case CORPORATE_ENTITY:
+                target.setKind("corporate-entity-person-with-significant-control");
+                break;
+            case LEGAL_PERSON:
+                target.setKind("legal-person-person-with-significant-control");
+                break;
+            case SUPER_SECURE:
+                target.setKind("super-secure-person-with-significant-control");
+                break;
+            case INDIVIDUAL_BENEFICIAL_OWNER:
+                target.setKind("individual-beneficial-owner");
+                break;
+            case CORPORATE_BENEFICIAL_OWNER:
+                target.setKind("corporate-entity-beneficial-owner");
+                break;
+            case LEGAL_PERSON_BENEFICIAL_OWNER:
+                target.setKind("legal-person-beneficial-owner");
+                break;
+            case SUPER_SECURE_BENEFICIAL_OWNER:
+                target.setKind("super-secure-beneficial-owner");
+                break;
+            default:
+        }
+    }
+
+    /**
+     * Manually map CeasedOn.
+     * @param target Data object within FullRecordCompanyPSCApi object to map to
+     * @param source Psc delta object that will be mapped from
+     */
+    @AfterMapping
+    default void mapCeasedOn(@MappingTarget Data target, Psc source) {
+        if (source.getCeasedOn() != null) {
+            target.setCeasedOn(MapperUtils.parseLocalDate(source.getCeasedOn()));
+        }
+    }
+
+    /** encode psc_id. */
+    @AfterMapping
+    default void mapEncodedPscId(@MappingTarget ExternalData target, Psc source) {
+        if (source.getPscId() != null) {
+            target.setPscId(MapperUtils.encode(source.getPscId()));
+        }
+    }
+
+    /**
+     * Manually map ServiceAddressSameAsRegisteredOffice.
+     * @param target Data object within FullRecordCompanyPSCApi object to map to
+     * @param source Psc delta object that will be mapped from
+     */
+    @AfterMapping
+    default void mapRegisteredOfficeEnum(@MappingTarget Data target, Psc source) {
+        Psc.ServiceAddressSameAsRegisteredOfficeEnum registeredOfficeEnum =
+                source.getServiceAddressSameAsRegisteredOffice();
+
+        if (registeredOfficeEnum != null) {
+            if (registeredOfficeEnum.toString().equals("Y")) {
+                target.setServiceAddressSameAsRegisteredOfficeAddress(Boolean.TRUE);
+            } else if (registeredOfficeEnum.toString().equals("N")) {
+                target.setServiceAddressSameAsRegisteredOfficeAddress(Boolean.FALSE);
+            }
+        }
+    }
+
+    /**
+     * Manually map ResidentialAddressSameAsServiceAddress.
+     * @param target Data object within FullRecordCompanyPSCApi object to map to
+     * @param source Psc delta object that will be mapped from
+     */
+    @AfterMapping
+    default void mapServiceAddressEnum(@MappingTarget SensitiveData target, Psc source) {
```

Comment:
I think this should be called mapResidentialAddressSameAsServiceAddress for clarity